### ISSUE TITLE
Error in min-max installs in UK

### ISSUE THREAD

**Issue by blanxii** [facundoolano/google-play-scraper#107]
Hi,

We're trying to fetch country: UK lang: EN and we have some problems.

```
  const installs = additionalInfo.find('div.content[itemprop="numDownloads"]').text().trim().split(' - '); <------------ Result: "1 000 000–5 000 000"
  const minInstalls = cleanInt(installs[0]);
  const maxInstalls = cleanInt(installs[1]) || undefined;
```

The problem was the character "-" is different for uk and es 

```
uk: –
es: -
```

Thanks!

**Comment by facundoolano** [comment]
It looks like we'll need to use a regex in the `split` to consider both cases. Can you show me the request you are making so I can debug the issue?

**Comment by blanxii** [comment]
So fast answer!! Thanksss


```
gplay.app({
  appId: 'com.sharkparty.monopolybingo',
  lang: 'en',
  country: 'uk'
})

```

**Comment by jlopezxs** [comment]
What is the status of this bug?
Thanks!

**Comment by facundoolano** [comment]
haven't had a chance to look at it yet.

**Comment by facundoolano** [comment]
I'm not able to reproduce the issue:

```js
> gplay.app({appId: 'com.sharkparty.monopolybingo', lang: 'en', country: 'uk'}).then((app) => console.log(app.minInstalls, app.maxInstalls))
> 5000000 10000000
```
Does that fail for you? Can you show me another example that does fail?

**Comment by facundoolano** [comment]
Actually now that I look at the code you pasted in the first message, you seem to be using an outdated  version of the module. The installs issue has bee fixed recently so try with 3.0.0

Closing this, open a new issue if you still have problems.
